index.html

```diff
@@ -1,10 +1,41 @@
 <!DOCTYPE html>
 <html lang="en">
+<h1>Моя первая страница</h1>
 <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" />
     <title>Title</title>
-</head>
-<body>
+  </head>
+  <body>
+    <blockquote cite="https://www.interface.ru/home.asp?artId=25273">
+        Измерять продуктивность программиста подсчетом строк кода — это так же, как оценивать постройку самолета по его весу.
+        <strong>Bill Gates</strong>.
+    </blockquote>
+    <hr>
+    <h2> Для того, чтобы стать фронт энд-разработчиком нужно выучить:</h2>
 
-</body>
-</html>
\ No newline at end of file
+         <h3><em>Html и Css</em></h3>
+        Именно эти языки отвечают за то, как сайт будет выглядеть в браузере. Выучить их можно <del>самостоятельно с помощью интернет-ресурсов</del>. <mark>Главное</mark> - сразу применять на практике, создавая подобия сайтов. Самое главное на этом этапе - не выучить все наизусть, а получить представление о языках, их работе, атрибутах и возможностях. Когда вы будете понимать, что вам конкретно нужно, вы сможете быстро загуглить нужный тег, важнее понимать структуру в целом.
+        <h3><em>Верстка макетов</em></h3>
+        Вы можете найти гайды по верстке в интернете и шаг за шагом учиться превращать PSD-макеты в веб-страницы.
+        На этом этапе вы уже можете пробоваться на должность HTML-верстальщик.
+        <h3><em>JavaScript</em></h3>
+        Это язык программирования, который позволяет вам создать динамически обновляемый контент, управляет мультимедиа, анимирует изображения.
+
+      <h2>Советы:</h2>
+    <p>
+        1 Копайте глубже <br>
```

Comment:
Better use [ordered list](https://developer.mozilla.org/ru/docs/Web/HTML/Element/ol) here.

Reply:
Ok, but such type of elements we were explained in Lesson 3.

Reply:
On your choice

Reply:
I'll do it in next task